samples/net/zoap_server/prj.conf

```diff
@@ -9,3 +9,4 @@ CONFIG_TEST_RANDOM_GENERATOR=y
 CONFIG_ZOAP=y
 CONFIG_NET_APP_SETTINGS=y
 CONFIG_NET_APP_MY_IPV6_ADDR="2001:db8::1"
+CONFIG_NET_IF_MCAST_IPV6_ADDR_COUNT=5
```

Comment:
I took a better look at this and found out that:
* Each multicast address takes 20 bytes ram, so we increase the ram consumption by 40 bytes if we change the default from 3 to 5. This is a small number but these things seem to add up.
* There is no "safe" value here. The current default 3 is usually quite ok for normal cases, but for some protocols like CoAP that use multicast a lot, that might be too small.

So after second though I suggest we leave the current default to 3 and just increase the number in samples where it makes sense.